cmd/noms/noms_log.go

```diff
@@ -99,8 +100,12 @@ func runLog(args []string) int {
 	defer pgr.Stop()
 
 	for commitBuff := range outChan {
-		io.Copy(pgr.Writer, bytes.NewReader(commitBuff.([]byte)))
+		_, err := io.Copy(pgr.Writer, bytes.NewReader(commitBuff.([]byte)))
+		if err != nil {
+			os.Exit(0)
```

Comment:
Here are the stacks when I use `break` in case you're interested:

https://gist.github.com/ahl/35937d020e585d6d3aa28f638dc46273